### DISCUSSION TITLE
Nudge amount

### DISCUSSION THREAD

**Discussion by addisonjames** [rxhanson/RectanglePro-Community#279] [Ideas]
The current nudge amount is very small, seems about 20px. Can this be adjustable? I'd like it to be more like 100px - 200px. Another alternative is to add a "small nudge" and a "big nudge".
Thanks

**Comment by rxhanson** [comment]
Indeed, 20px is the default.

You can change it with this terminal command, followed by an app restart:

`defaults write com.knollsoft.Hookshot nudgeDistance -float 100`

**Reply by stoobe** [reply]
This is perfect, thanks! Is there a similar setting for zoom amount or percent? I'd like to adjust that too.

**Reply by rxhanson** [reply]
30px is the default for that.

```
defaults write com.knollsoft.Hookshot sizeOffset -float <NUM_PIXELS>
```

Nearly all of these are usable for Rectangle Pro: https://github.com/rxhanson/Rectangle/blob/main/TerminalCommands.md

Just replace `com.knollsoft.Rectangle` with `com.knollsoft.Hookshot`.
